docs/orchestrating-elastic-stack-applications/agent-standalone.asciidoc

```diff
@@ -454,8 +457,104 @@ spec:
   deployment:
     podTemplate:
       spec:
+        securityContext:
+          # Gid 1000 is the default group at which the Agent container runs.
+          # Adjust as necessary if `runAsGroup` has been modified.
+          fsGroup: 1000
+        # runAsUser: 0 can be disabled when running with an `emptyDir`
+        # as opposed to a `hostPath` volume.
+        # securityContext:
+          # runAsUser: 0
         volumes:
         - name: agent-data
           emptyDir: {}
 ...
+----
+
+
+[source,yaml]
+----
+# Using a DaemonSet to manage permissions for Elastic Agent using hostPath volume.
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: elastic-agent
+  namespace: elastic-apps
+spec:
+  daemonSet: {}
+...
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: manage-agent-hostpath-permissions
+  namespace: elastic-apps
+spec:
+  selector:
+    matchLabels:
+      name: manage-agent-hostpath-permissions
+  template:
+    metadata:
+      labels:
+        name: manage-agent-hostpath-permissions
+    spec:
+      # This is only required when running in an selinux-enabled/Openshift environment.
+      # Ensure this user has been added to the privileged scc in the correct namespace.
+      # oc adm policy add-scc-to-user privileged -z elastic-agent -n elastic-apps
+      # serviceAccountName: elastic-agent
+      volumes:
+        - hostPath:
+            path: /var/lib/elastic-agent
+            type: DirectoryOrCreate
+          name: "agent-data"
+      initContainers:
+        - name: manage-agent-hostpath-permissions
+          # fedora is only required when needing the `chcon` binary when running
+          # in an senlinux-enabled/Openshift environment. If that
+          # is not required then the following smaller image can be used instead:
+          # image: fedora:39
```

Comment:
I think `chcon` is also available in `registry.access.redhat.com/ubi8/ubi-minimal:latest` which would avoid the static version, and "feels" lighter than "fedora" (only a gut feeling I did not check 😄 )